invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/FileUploader/FileUploader.js

```diff
@@ -132,8 +132,8 @@ export const FileUploaderComponent = ({
                   {humanReadableBytes(
                     filesSize + acceptedFilesSize,
                     decimalSizeDisplay
-                  )}
-                  {i18next.t("but the limit is")}
+                  )}{" "}
+                  {i18next.t("but the limit is")}{" "}
```

Comment:
Maybe we should wrap that whole sentence in one <Trans> so translators get full context and we lose the hacky spaces as in the docs here: https://inveniordm.docs.cern.ch/develop/howtos/i18n

Something like:
```jsx
<Message
  warning
  icon="warning circle"
  header={i18next.t("Could not upload files.")}
  content={
    <Trans
      i18nKey="fileUploadQuotaExceeded"
      values={{
        total: humanReadableBytes(
          filesSize + acceptedFilesSize,
          decimalSizeDisplay
        ),
        limit: humanReadableBytes(quota.maxStorage, decimalSizeDisplay),
      }}
    >
      Uploading the selected files would result in {{ total }} but the limit is
      {{ limit }}.
    </Trans>
  }
/>
```

WDYT?

Reply:
I think that looks a lot cleaner. I'd also propose changing the english to 

`Uploading the selected files would result in {{ total }} of storage use but the limit is
      {{ limit }}`

Reply:
I would go with the following :)
`Uploading the selected files would result in {{ total }} of storage use, exceeding the limit of {{ limit }}`

And agree Sam, but would need to be tested to check and I don't have time at current

Reply:
nevermind, done!